save-agent/src/linuxX64Test/kotlin/com/saveourtool/save/agent/SaveAgentTest.kt

```diff
@@ -20,14 +19,17 @@ import kotlin.test.assertEquals
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.runBlocking
-import kotlinx.serialization.ExperimentalSerializationApi
 import kotlinx.serialization.PolymorphicSerializer
-import kotlinx.serialization.properties.Properties
-import kotlinx.serialization.properties.decodeFromStringMap
 
 class SaveAgentTest {
-    @OptIn(ExperimentalSerializationApi::class)
-    private val configuration: AgentConfiguration = Properties.decodeFromStringMap<AgentConfiguration>(readProperties("src/linuxX64Main/resources/agent.properties")).let {
+    init {
+        setenv(AgentEnvName.AGENT_ID.name, "agent-for-test", 1)
+        setenv(AgentEnvName.BACKEND_URL.name, "http://localhost:5800", 1)
```

Comment:
are you sure that we should hardcode port here?

Reply:
It's a test, this port doesn't even matter